checkersfinal.py

```diff
@@ -1,123 +1,43 @@
-            string_to_add = ""
-    
-    rows_to_print.append("- White Origin -")
-    for i in rows_to_print:
-        print(i)
-        
-    # for row in range(game.board.height):
-    #     for column in range(game.board.width):
-    #         current_position = game.board.position_layout[row][column]
-    #         print(current_position)
-    #         if game.board.position_is_open(current_position):
-    #             print("open position")
-
-# NOT CONVERTED FOR V2 BOARD
-# simulated play between two CPU players
-# current_game - the object on which the game is running on
-def simulated_play(current_game):
-    # assume player 1 is black
-    # assume player 2 is white
-    game_over = False
-    current_player = 0 # for now, first player will be Black
-    current_iteration = 1
-    last_move = ""
-    while game_over == False:
-        draw_iteration_details(current_game,current_iteration)
-
-        draw_board(current_game)
-
-        possible_moves = current_game.get_possible_moves()
-        random_number = randint(0, len(possible_moves)-1)
-
-        last_move = str(possible_moves[random_number])
-
-        current_game.move(possible_moves[random_number])
-
-        print()
-
-        game_over =  current_game.is_over()
-    
-    
-    print("--------------------------")
-    
-    print("Final Iteration: " + str(current_iteration))
-    draw_board(current_game)
-
-    print_winner(current_game)
-
-# checkers game between a person and a CPU
-def human_vs_cpu_play(current_game,pygame_instance = None):
+import alphabeta
+
+ALPHABETA = 'alphabeta'
+RANDOM = 'random'
+
+def human_vs_alphabeta_no_gui(board):
+    # Use this function to play without the pygame gui
+    # Loop while we don't have a winner
+    while not board.has_winner():
+        # First get user input for white
+        board.print_board()
+        valid_moves = board.print_all_valid_moves()
+        print(valid_moves)
+        print("Select a piece to move:")
+        piece_row = int(input("row: "))
+        piece_col = int(input("column: "))
+        if (piece_row, piece_col) not in valid_moves:
+            print("Not a valid piece")
+            continue
+        print("Select move to make (by index): ")
+        moves_for_piece = valid_moves[piece_row, piece_col]
+        print(moves_for_piece)
+        move_index = int(input())
+        if move_index > len(moves_for_piece):
+            print("Not a valid move index")
+            continue
+        move = moves_for_piece[move_index][1]
+        board.make_moves([(piece_row, piece_col), (move[0], move[1])])
+        board.print_board()
+
+        # Now let our player play for black
+        print("Thinking....")
+        alphabeta.play_move(board)
+
+# checkers game between a person and a random player
+def human_vs_random_play(current_game, pygame_instance = None, play_strategy = ALPHABETA):
     # allow keyboard input
     # make some sort of cli interface
     # determine who goes first
```

Comment:
maybe add if player want to do alphabeta or random chice
`play_type = input("Will computer make (1)random or (2)AlphaBeta move: ")

if play_type == 1:
        play_strategy = RANDOM
else:
        play_strategy = ALPHABETA
`